src/Sets/VPolygon.jl

```diff
@@ -27,6 +27,41 @@ computation of the convex hull.
 - `VPolygon(vertices::Vector{Vector{N}};
             apply_convex_hull::Bool=true,
             algorithm::String="monotone_chain")`
+
+### Examples
+
+A polygon in vertex representation can be constructed by passing the list of
+vertices. For example, we can build the right triangle
+
+```jldoctest polygon_vrep
+julia> P = VPolygon([[0, 0], [1, 0], [0, 1]])
+VPolygon{Int64,Array{Int64,1}}(Array{Int64,1}[[0, 0], [1, 0], [0, 1]])
+
+julia> eltype(P)
+Int64
+```
+The element type of the polygon canbe changed by using a different vector type:
+
+```jldoctest polygon_vrep
+julia> P = VPolygon([[0, 0], [1, 0], [0, 1.]]) # mind the dot in [0, 1.]
+VPolygon{Float64,Array{Float64,1}}(Array{Float64,1}[[0.0, 0.0], [1.0, 0.0], [0.0, 1.0]])
+
+julia> eltype(P)
+Float64
+```
```

Comment:
I think this is too much spam for the type documentation. This is not a Julia tutorial.